mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -159,7 +159,9 @@ private[classification] trait LogisticRegressionParams extends ProbabilisticClas
 @Since("1.2.0")
 @Experimental
 class LogisticRegression @Since("1.2.0") (
-    @Since("1.4.0") override val uid: String)
+    @Since("1.4.0") override val uid: String,
+    @Since("2.0.0") val numFeatures: Int = 0,
```

Comment:
I think the values are passed in when user create the object. When user call toString, the values are returned.